frontend/src/core/app/router.js

```diff
@@ -86,6 +86,12 @@ define(function(require) {
         return $('.app-inner').append(new PermissionsView().$el);
       }
 
+      // Verify the user is authenticated
+      if (!this.isUserAuthenticated()  && (module !== 'user' && route1 !== 'login')) {
+        alert('Your session has expired, click OK to log on again');
```

Comment:
I agree that it's 'bleh', but thought leaving it as an alert() would aid the re-factoring when we revisit the entire error handling.